scipy/stats/morestats.py

```diff
@@ -2167,24 +2229,37 @@ def ansari(x, y):
     uxy = unique(xy)
     repeats = (len(uxy) != len(xy))
     exact = ((m < 55) and (n < 55) and not repeats)
-    if repeats and (m < 55 or n < 55):
+    if mode == 'exact' and repeats:
+        raise ValueError("can't compute exact statistic with ties. "
+                         "Please use `mode='auto'` or `mode='approx'`.")
```

Comment:
This is not what was described in the documentation. It wouldn't be backwards compatible, right?

Reply:
> This is not what was described in the documentation

Ah, yes. Sorry. But as I have commented above, once we decide what we should do when ties are present, I will update this and relevant tests.

> It wouldn't be backwards compatible, right?

Everything should be backward compatible as `mode='auto'` is the default and the behavior there has not been changed.

Reply:
I see.